src/js/api/setup-steps.js

```diff
@@ -18,7 +19,8 @@ export function loadPlaylist(_model) {
             });
             playlistLoader.on(ERROR, err => {
                 setPlaylistAttributes(_model, [], {});
-                reject(new Error(`Error loading playlist: ${err.message}`));
+                err.code = (err.code || 0) + SETUP_ERROR_LOADING_PLAYLIST;
```

Comment:
Do we need the `|| 0` ? `err.code += SETUP_ERROR_LOADING_PLAYLIST;`
Also we need to set the error's message to ``Error loading playlist: ${err.message}``

Reply:
If the code is undefined adding another number will make it `NaN`